easybuild/easyblocks/o/openblas.py

```diff
@@ -13,8 +14,31 @@ class EB_OpenBLAS(ConfigureMake):
     def build_step(self):
         """Custom build procedure for OpenBLAS."""
 
+        self.cfg['buildopts'] += 'BINARY=64 USE_THREAD=1 USE_OPENMP=1 CC="%s" FC="%s"' % (
+            os.environ['CC'], os.environ['F77'])
```

Comment:
This should rather be FC and not F77. Use of F77 should be avoided when possible.
Especially since it's used to set "FC"

Reply:
Yes, sorry! Copy-paste mistake I think.

Reply:
Well, all the existing easyconfigs for OpenBLAS also uses F77 so not your fault :-)